Doc/library/argparse.rst

```diff
@@ -620,27 +620,19 @@ keyword argument::
 color
 ^^^^^
 
-By default, the help message is printed in plain text. If you want to allow
-color in help messages, you can enable it by setting ``color`` to ``True``::
+By default, the help message is printed in color using `ANSI escape sequences
+<https://en.wikipedia.org/wiki/ANSI_escape_code>`__.
+If you want plain text help messages, you can disable this :ref:`in your local
```

Comment:
We should add a note here about this respecting NO_COLOR.

Reply:
Sure, although it links to `using-on-controlling-color`

Reply:
Sure, although it links to https://docs.python.org/3.14/using/cmdline.html#using-on-controlling-color which covers all the env vars: NO_COLOR, FORCE_COLOR, PYTHON_COLORS, plus the TERM stuff.

The wording was:

> This can be controlled by [environment variables](https://docs.python.org/3.14/using/cmdline.html#using-on-controlling-color)

And updated in https://github.com/python/cpython/pull/136809/commits/82c7e2511c631ab9dfb7a031bd078a27296233fa.

Would you like to suggest some wording?